packages/client/src/StreamrClient.ts

```diff
@@ -285,7 +285,17 @@ export class StreamrClient {
          */
         messageMatchFn?: (msgTarget: Message, msgGot: Message) => boolean
     }): Promise<void> {
-        return this.resends.waitForStorage(message, options)
+        const defaultOptions = {
+            // eslint-disable-next-line no-underscore-dangle
+            interval: this.config._timeouts.storageNode.retryInterval,
+            // eslint-disable-next-line no-underscore-dangle
+            timeout: this.config._timeouts.storageNode.timeout,
+            count: 100,
+            messageMatchFn: (msgTarget: Message, msgGot: Message) => {
+                return msgTarget.signature === msgGot.signature
+            }
```

Comment:
Given that this is the only implementation in use in our code base, should it just be hard coded into `waitForStorage`?